packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx

```diff
@@ -87,6 +90,19 @@ const CollapsibleTradeParams = ({
                 {is_multiplier && <MultiplierOptionsWidget />}
                 {isVisible('trade_type_tabs') && <TradeTypeTabs />}
                 {is_accumulator && <AccumulatorOptionsWidget />}
+                {isVisible('last_digit') && is_collapsed && (
```

Comment:
Not sure if I should make a separate component for it as it's only `<Text>` and `<Localize>`